app/external_bounties/management/commands/sync_bounty0x.py

```diff
@@ -43,6 +43,7 @@ def handle(self, *args, **options):
 
         response = requests.get(url, headers=headers)
         bounties = response.json()
+        print(len(bounties))
```

Comment:
@owocki  take this away ?

Reply:
da73e3cd79cbdc8e7114e8bdde8577571b455a68